### PR TITLE
Update compiling-rpms.md

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Added the steps for building rpm packages in CentOS 8.x

### CODE REVIEW

File: docs/Developer-guide/compiling-rpms.md [hunk 1]
```diff
@@ -6,17 +6,18 @@ Creating rpm's of GlusterFS from git source is fairly easy, once you know the st
 RPMS can be compiled on at least the following OS's:
 
 - Red Hat Enterprise Linux 5, 6 (& 7 when available)
-- CentOS 5, 6 (& 7 when available)
+- CentOS 5, 6, and 8
```
Comment (kshithijiyer) [inline docs/Developer-guide/compiling-rpms.md:9]: Why are  we removing CentOS 7 from the list? Is it no longer supported?

File: docs/Developer-guide/compiling-rpms.md [hunk 2]
```diff
@@ -73,7 +93,7 @@ You'll need EPEL installed first and some RHEL specific packages. The 2 commands
 		# sudo yum -y --enablerepo=rhel-6-server-optional-rpms install python-webob1.0 \
 	        python-paste-deploy1.5 python-sphinx10 redhat-rpm-config
 
-Now follow through the **Common Steps** part below.
+Now follow through the **Common Steps** part below.
```
Comment (kshithijiyer) [inline docs/Developer-guide/compiling-rpms.md:96]: Do we really need to add this space?

File: docs/Developer-guide/compiling-rpms.md (hunk 1 above)
Comment (srijan-sivakumar) [reply docs/Developer-guide/compiling-rpms.md:9]: No..no.. I must have removed it by mistake while editing it. My bad.

File: docs/Developer-guide/compiling-rpms.md (hunk 2 above)
Comment (srijan-sivakumar) [reply docs/Developer-guide/compiling-rpms.md:96]: not required. Ack. will remove it.

**Review by rkothiya — COMMENTED** [review]
LGTM

**Pull request merged**